components/portal/timeout/controllers.js

```diff
@@ -21,12 +21,24 @@
 define(['angular'], function(angular) {
   return angular.module('portal.timeout.controllers', [])
   .controller('PortalTimeoutController', [
-    '$sessionStorage', '$log', '$location', '$timeout',
-    '$mdDialog', '$window', 'MISC_URLS', 'portalShibbolethService',
+    '$sessionStorage', '$log', '$timeout', '$interval',
+    '$mdDialog', '$window', 'APP_OPTIONS', 'MISC_URLS',
+    'portalShibbolethService', 'sessionInactivityService',
   function(
-    $sessionStorage, $log, $location, $timeout,
-    $mdDialog, $window, MISC_URLS, portalShibbolethService
+    $sessionStorage, $log, $timeout, $interval,
+    $mdDialog, $window, APP_OPTIONS, MISC_URLS,
+    portalShibbolethService, sessionInactivityService
   ) {
+    var checkInactiveDelay = 1; // Minutes
+    var appInactiveTimeout = APP_OPTIONS.inactivityTimeout; // Minutes
```

Comment:
Consider documenting `APP_OPTIONS.inactivityTimeout` in [configuration.md](https://github.com/uPortal-Project/uportal-app-framework/blob/master/docs/configuration.md#app_options)